src/ConfirmedMap.js

```diff
@@ -20,11 +21,41 @@ class MbMap extends React.Component {
         };
     }
 
+    // Check if a date was more than two weeks ago
+    isOld(date) {
+        // Working with raw data, so try-catch just in case
+        try {
```

Comment:
haha this is also what I thought, just push the whole process into a try function